operations/mimir/ruler-remote-evaluation.libsonnet

```
@@ -70,6 +74,7 @@
 
       // The ruler-query-frontend sends the query response back to the ruler.
       'server.grpc-max-send-msg-size-bytes': $._config.ruler_remote_evaluation_max_query_response_size_bytes,
+      'ruler.client.grpc-max-recv-msg-size': $._config.ruler_remote_evaluation_max_query_response_size_bytes,
```

Queriers call an RPC on query-frontends to send the result, so we need to use the `server` config for this:

```suggestion
      'server.grpc-max-recv-msg-size-bytes': $._config.ruler_remote_evaluation_max_query_response_size_bytes,
```

We should probably also add a comment above to separate this from the comment about ruler-query-frontend sending the response to rulers.